client/setup.py

```diff
@@ -43,6 +43,7 @@
     "httpx>=0.22.0",
     "urllib3<1.27",
     "pydantic<2.0.0",  # current broker and fastapi lib code only work with pydantic < 2.0.0
+    "fire",
```

Comment:
`fire` and `click`(current use for command line parser) are similar libs, we maybe use only one lib. The mixed usage maybe confused.

Reply:
done